routes/events/events.js

```diff
@@ -1,4 +1,5 @@
 const express = require('express');
+const Fuse = require('fuse.js');
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module 'fuse.js'.